extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -52,10 +62,22 @@
     @ConfigRoot(name = "smallrye-health")
     static final class SmallRyeHealthConfig {
         /**
-         * The path of the health-checking servlet.
+         * The path of the legacy health-checking servlet.
```

Comment:
I think we should change it to say that it is the root path for the SmallRye Health servlet.

It being the path of the legacy servlet is a direct consequence of it and should be hidden, considered it's just a legacy behavior.